back/src/domain/convention/useCases/SignConvention.ts

```diff
@@ -53,21 +57,24 @@ export class SignConvention extends TransactionalUseCase<
   }
 
   public async _execute(
-    _: void,
+    { conventionId }: WithConventionId,
     uow: UnitOfWork,
-    { applicationId, role }: ConventionJwtPayload,
+    params: ConventionDomainPayload | InclusionConnectDomainJwtPayload,
```

Comment:
le params m'a induit en erreur. On peut renomer, par exemple `jwtPayload` ?
```suggestion
    jwtPayload: ConventionDomainPayload | InclusionConnectDomainJwtPayload,
```